src/components/CommandBar/CommandsList/index.tsx

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
prefer doing this on top of statements, not on top of file. to know exactly what we're disabling

Reply:
It still shows the error when using for `next-line` so had to resort to this.